servers/src/test/java/tachyon/master/next/filesystem/meta/InodeDirectoryTests.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.next.filesystem.meta;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tachyon.Constants;
+import tachyon.master.block.BlockId;
+
+/**
+ * Unit tests for tachyon.InodeDirectory
+ */
+public final class InodeDirectoryTests {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+
+  @Test
+  public void addChildrenTest() {
+    InodeDirectory inodeDirectory =
```

Comment:
`inodeDirectory` is created almost in the same way in each test case, you can make it a member variable of the tests.